dataprofiler/tests/profilers/test_utils.py

```diff
@@ -44,3 +44,42 @@ def test_uneven_chunk_sample(self):
             num_chunks += 1
         self.assertEqual(num_chunks, 100 // 7 + 1)
         self.assertCountEqual(all_values, list(range(100)))
+        
+    def test_find_diff(self):
+        """
+        Checks to see if the find difference function is operating
+        appropriately.
+        """
+
+        # Ensure lists and sets are handled appropriately
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets([3, 2], [2, 3, 2]))
+        self.assertEqual([[1], [2, 3], [4]],
+                         utils.find_diff_of_lists_and_sets([1, 2, 3], [2, 3, 4]))
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets({3, 2}, {2, 3}))
+        self.assertEqual([[1], [2, 3], [4]],
+                         utils.find_diff_of_lists_and_sets({1, 2, 3}, {2, 3, 4}))
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets({2, 3}, [2, 3]))
+        self.assertEqual([[1], [2, 3], [4]],
+                         utils.find_diff_of_lists_and_sets([1, 2, 3], {2, 3, 4}))
```

Comment:
Is there ever a case where these functions would get called with 2 things of different types?

Reply:
Don't know but I was treating sets and lists the same so I thought they ought to be interchangeable too. I think there are some parts of the repo that go from lists to sets, so maybe its possible